setup.py

```diff
@@ -84,10 +84,13 @@
     "invenio-userprofiles>=1.0.1,<1.1.0",
     # extra
     "invenio-circulation>=1.0.0a19,<1.1.0",
+    "invenio-stats>=1.0.0a12",
     # until flask-sqlalchemy is fixed
     "SQLAlchemy>=1.2.16,<1.3.0",
     "invenio-pidrelations>=1.0.0a6,<1.1.0",
     "sentry-sdk>=0.10.2",
+    # kombu 4.6.5 requires amqp==2.5.1
+    "amqp==2.5.1",
```

Comment:
don't need anymore?

Reply:
I will remove it, but maybe this is the reason it was passing, we ll see